src/distribution/laplace.rs

```diff
@@ -0,0 +1,386 @@
+use distribution::{Continuous, Univariate};
+use rand::distributions::Distribution;
+use statistics::*;
+use rand::Rng;
+use rand::distributions::OpenClosed01;
+use std::f64;
+use {Result, StatsError};
+
+/// <summary>
```

Comment:
I'm not sure rustdoc can handle doc comments like this, but it's good to be consistent with the rest of the code base. Please refer to [the rustdoc docs](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html) or [another distribution](https://github.com/boxtown/statrs/blob/master/src/distribution/binomial.rs#L9) for examples

Reply:
Fixed it up!